sim/main.py

```diff
@@ -0,0 +1,246 @@
+"""
+main.py
+
+Author: Matthew Yu, Array Lead (2020).
+Contact: matthewjkyu@gmail.com
+Created: 5/24/20
+Last Modified: 5/28/20
+Description: This is the main handler for the mppt simulator. 
+  It uses Kivy for the UI of this application, in particular the graphs and statistics.
+  The simulator has three main components:
+      1. Source simulator - The program simulates a solar cell or set of solar cells by taking in static parameters or a datafile of input values over time.
+      2. MPPT Algorithm - A black box that manages algorithms (PandO, Incr. Conduction, RCC, Fuzzy Logic, etc) to maximize power on the load.
+      3. Simulator - The outputs of the MPPT and the source are calculated, stored, and graphed. Data over time will be evaluated to determine and compare efficiencies between algorithms.
+"""
+from source import Source
+from simulation import Simulation
+
+from mppt_algorithms.mppt_perturb_and_observe import PandO
+from mppt_algorithms.mppt_incremental_conduction import IC
+# from mppt_algorithms.mppt_ripple_correlation_control import RCC
+# from mppt_algorithms.mppt_fuzzy_logic import FL
+from mppt_algorithms.mppt_dP_dV_feedback_control import FC
+
+def main():
```

Comment:
When I tried running this, `python main.py` didn't seem to work. It took a moment to realize that this only works with python3. I suggest adding something like
```python
if sys.version_info[0] < 3:
    raise Exception("Must be using Python 3")
```

This requires importing `sys`.